packages/tina-graphql-gateway/src/auth/TinaCloudProvider.tsx

```diff
@@ -36,7 +37,10 @@ export interface TinaCloudAuthWallProps {
   getModalActions?: (args: {
     closeModal: () => void
   }) => { name: string; action: () => Promise<void>; primary: boolean }[]
-  mediaStore?: TinaCloudMediaStoreClass
+  mediaStore?: {
```

Comment:
It might be Nice if this had the type `MediaStore || {store: MediaStore, pageSize: MediaStore}`. One other thing is that the user can also set this up on there own by passing there own CMS object